src/components/LabelComment.js

```diff
@@ -0,0 +1,82 @@
+
+import React, { useState } from 'react';
+import * as Showdown from "showdown";
+import ReactMde from "react-mde";
+
+// import TextareaAutosize from 'react-textarea-autosize';
+import style from './LabelComment.module.scss';
+import './react-mde-all.css'
+
+const LabelComment = props => {
+
+    
+    const loadSuggestions = text => {
+        return new Promise((accept, reject) => {
+            
+        });
+    }
+
+    const converter = new Showdown.Converter({
+        tables: true,
+        simplifiedAutoLink: true,
+        strikethrough: true,
+        tasklists: true
+    });
+
+    const [profileState] = useState(props);
+
+    
+    const [value, setValue] = React.useState("**Comentário**");
+    const [selectedTab, setSelectedTab] = React.useState("write");
+    return (
+        <div>
+            <label>
+                <h4> {profileState.title} </h4>
+
+                <div className={profileState.style[1]}>
+
+                    <ReactMde
+                        className={profileState.style[0]}
+                        value={value}
+                        onChange={setValue}
+                        selectedTab={selectedTab}
+                        minEditorHeight="99px"
+                        minPreviewHeight="8000px"
+                        maxEditorHeight="2200px"
+                        onTabChange={setSelectedTab}
+                        generateMarkdownPreview={markdown =>
+                            Promise.resolve(converter.makeHtml(markdown))
+                        }
+                        
+                        childProps={{
+                            writeButton: {
+                                tabIndex: -1
+                            }
+                        }}
+                        classes={{
```

Comment:
ese tipo de fazer o css, está de acordo com todos os nossos? porq eu n fiz isso, se tiver de acordo, eu fiz errado xD

Reply:
ta assim pq ele usou o material UI nesse componente...
Eu sei que o Eduardo usou tbm.. o que vc sugere fazer @eboliveira ?
acho que não cabe mudar isso agr.. mas não sei :thinking:

Reply:
se não for mudar.. useMemo aqui

Reply:
Usei useMemo pra organizar